tests/collections/nlp/test_nlp_exportables.py

```diff
@@ -221,7 +227,7 @@ def test_QAModel_export_to_onnx(self):
         model = nemo_nlp.models.QAModel.from_pretrained(model_name="qa_squadv2.0_bertbase")
         with tempfile.TemporaryDirectory() as tmpdir:
             filename = os.path.join(tmpdir, 'qa.onnx')
-            model.export(output=filename, check_trace=True)
+            model.export(output=filename, check_trace=True, use_dynamo=True)
```

Comment:
Are we only testing dynamo-based ONNX export with this change? Is there value in testing both export paths i.e., TorchScript and Dynamo

Reply:
Well there definitely is. Maybe a good balance between snowballing test time and test coverage would be testing both options of export in the same test method (so sharing model initialization).